pages/_document.js

```diff
@@ -38,6 +38,10 @@ class MyDocument extends Document {
           <meta property="og:image" content="https://cdn.chec.io/email/assets/marketing/demo-preview.png" />
           <meta property="og:description" content="A custom, open source demo storefront built with Next.js and Commerce.js. Take a look!" />
           <meta property="og:url" content="https://commercejs-demo-store.netlify.app" />
+          <meta property="twitter:title" content="Commerce.js | Demo storefront" />
```

Comment:
```suggestion
          <meta property="twitter:title" content="Commerce.js | Demo storefront" />
          <meta name="twitter:creator" content="@commercejs">
```

May as well, right? :)